README.md

```diff
@@ -51,7 +51,31 @@ wskdeploy
 ```
 
 ### Bind Service Credentials
-You will need to bind your service credentials to the package, so that the Actions will have access to the service credentials. Individual README instructions within each package detail this step.
+You will need to bind your service credentials to the package, so that the Actions will have access to the service credentials. Individual README instructions within each package detail this step with specific commands to use per package.
```

Comment:
```markdown
Bind your service credentials to the package so that the Actions have access to the service credentials. For details and specific commands for each package, see the package README.